build.rs

```diff
@@ -202,6 +202,20 @@ fn main() {
         ccfg.flag_if_supported(flag);
     }
 
+    fn profile_config(cfg: &mut cc::Build) {
+        if cfg!(all(
+            feature = "profile",
+            target_os = "linux",
+            target_arch = "x86_64"
+        )) {
```

Comment:
Here I might suggest using the `cfg!` stanza to declare a `const bool`, which allows a bit more clarity about what the test is getting at:

```rust
const PLATFORM_CAN_PROFILE_WITHOUT_FRAME_POINTER: bool = cfg!(...);

if PLATFORM_CAN_PROFILE_WITHOUT_FRAME_POINTER { ... }
```

Or, you know, just a code comment to that effect.

Reply:
Ok, I added an any() around that all and a comment too, so it's obvious how to add more platforms.